kili/queries/asset/__init__.py

```diff
@@ -322,6 +327,9 @@ def count_assets(self, asset_id: Optional[str] = None,
                 },
                 'updatedAtGte': updated_at_gte,
                 'updatedAtLte': updated_at_lte,
+                'datasetAsset': {
```

Comment:
@mDuval1 is it a breaking change ? can you test it with the previous version of the playground ?

Reply:
No, not breaking